R/read_write.R

```diff
@@ -158,10 +157,17 @@ h5check <- function(fname, group, name) {
   bs <- adf( target_id = main_est$target_id )
   bs$est_counts <- as.numeric(rhdf5::h5read(fname, paste0("bootstrap/bs", i)))
   bs$tpm <- counts_to_tpm(bs$est_counts, main_est$eff_len)
-
   bs
 }
 
+read_bootstrap_mat <- function(fname, num_bootstraps, num_transcripts, est_count_sf) {
+  bs_mat <- matrix(nrow=num_bootstraps, ncol=num_transcripts)
+  for(i in 1:nrow(bs_mat)) {
+    bs_mat[i, ] <- rhdf5::h5read(fname, paste0("bootstrap/bs", i - 1)) / est_count_sf
```

Comment:
note to self: `est_count_sf` usage does not look correct here. I am fairly certain it should be `est_count_sf[i]`. should check for correctness